PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeCheckoutRequest.kt

```diff
@@ -7,5 +7,20 @@ data class PayPalNativeCheckoutRequest(
     /**
      * The order ID associated with the request.
      */
-    val orderId: String
+    val orderId: String,
+
+    /**
+     * Optional: The setup configuration for authentication
+     */
+    val authConfig: AuthConfig? = null
```

Comment:
```suggestion
    val email: String? = null
```
nit: would it be cleaner to just have a `val email: String?` property ?

Reply:
➕1 here

Reply:
I believe Kevin separated the `AuthConfig ` data class in the Checkout repo because there are a bunch of other properties on the Config object. I can update this to add the email field directly to request object here.